src/main/java/edu/kit/ipd/crowdcontrol/objectservice/mail/MailHandler.java

```diff
@@ -39,22 +40,39 @@ public MailHandler(Properties props, Authenticator auth) throws MessagingExcepti
         connectToStore();
```

Comment:
you open the resource, but i don't see where you close it. You have to be careful with opening resources!